src/test/java/io/vertx/core/spi/tracing/HttpTracerTestBase.java

```diff
@@ -218,7 +225,7 @@ public Object sendRequest(Context context, SpanKind kind, TracingPolicy policy,
       @Override
       public void receiveResponse(Context context, Object response, Object payload, Throwable failure, TagExtractor tagExtractor) {
         assertSame(val, context.getLocal(key));
-        assertTrue(context.removeLocal(key));
+        context.putLocal(key, null);
```

Comment:
For convenience, how about adding a `removeLocal(key)` method?

Reply:
I think I added it but I haven't used it